test/models/order_test.rb

```diff
@@ -1,6 +1,24 @@
 require "test_helper"
 
 class OrderTest < ActiveSupport::TestCase
+  should validate_presence_of(:first_name)
+  should validate_presence_of(:last_name)
+  should validate_presence_of(:address)
+  should validate_presence_of(:city)
+  should validate_presence_of(:state)
+  should validate_presence_of(:zipcode)
+  should validate_inclusion_of(:status). in_array(['ordered', 'paid',
+                                                    'completed','cancelled'])
```

Comment:
Align the elements of an array literal if they span more than one line.<br>Space missing after comma.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.